pontoon/base/views.py

```diff
@@ -208,15 +208,20 @@ def _get_all_entities(locale, project, form, entities):
     visible_entities = entities.values_list('pk', flat=True)
 
     return JsonResponse({
-        'entities': Entity.map_entities(locale, entities_to_map, visible_entities),
+        'entities': Entity.map_entities(
+            locale,
+            preferred_source_locale,
+            entities_to_map,
+            visible_entities
```

Comment:
Nit: please add comma at the end of the line. That code style allows us to add/remove parameters by making 1-line diffs.